src/components/WalletModal/Option.tsx

```diff
@@ -62,25 +65,26 @@ const IconWrapper = styled.div`
   `};
 `
 
-type OptionProps = {
-  connection: Connection
-  activate: () => void
-  pendingConnectionType?: ConnectionType
-}
-export default function Option({ connection, pendingConnectionType, activate }: OptionProps) {
-  const isPending = pendingConnectionType === connection.type
+export default function Option({ connection }: { connection: Connection }) {
+  const { activationState, tryActivation } = useActivationState()
+  const closeDrawer = useCloseAccountDrawer()
+  const activate = () => tryActivation(connection, closeDrawer)
+
+  const isSomeConnectorPending = activationState.status === ActivationStatus.PENDING
+  const isCurrentOptionPending = isSomeConnectorPending && activationState.connection.type === connection.type
   const isDarkMode = useIsDarkMode()
-  const content = (
+
+  return (
     <TraceEvent
       events={[BrowserEvent.onClick]}
       name={InterfaceEventName.WALLET_SELECTED}
       properties={{ wallet_type: connection.getName() }}
       element={InterfaceElementName.WALLET_TYPE_OPTION}
     >
       <OptionCardClickable
-        onClick={!pendingConnectionType ? activate : undefined}
-        clickable={!pendingConnectionType}
-        disabled={Boolean(!isPending && !!pendingConnectionType)}
+        onClick={activate}
+        clickable={!isSomeConnectorPending}
```

Comment:
Removing `disabled` was a good call, I still don't understand why we had both. Anyway, do you think removing `clickable` and sticking to native `disabled` would actually make more sense here? That way we don't even need to set `pointer-events: none` for clickable, because `disabled` will disable the button entirely.

Reply:
Yes that makes way more sense, will use disabled instead thank you.

Previously, clickable meant whether the hover state should activate, and disabled meant whether the opacity should be lowered (because when one connector is pending, all should _not_ be clickable, but the selected one should not have it's opacity lowered)

<img width="352" alt="image" src="https://user-images.githubusercontent.com/39385577/234907123-9513daa0-a88f-4335-ba6f-2364dacd29ff.png">

So we still have to pass in two vars to accomplish this, but I think we've arrived at the much easier-to-follow  `selected` and `disabled` compared to the previous `disabled` and `clickable`